ntc_templates/templates/cisco_xr_admin_show_inventory.textfsm

```diff
@@ -1,17 +1,15 @@
-Value NAME (.*)
+Value Required NAME (.*?)
 Value DESCR (.*)
-Value PID (([\S+]+|.*))
-Value VID (.*)
-Value SN ([\w+\d+]+)
+Value PID (\S+)
+Value VID (\S+)
+Value SN (\S*)
 
 Start
-  ^NAME:\s+"${NAME}",\s+DESCR:\s+"${DESCR}"
-  ^PID:\s+${PID}.*,.*VID:\s+${VID},.*SN:\s+${SN} -> Record
-  ^PID:\s+,.*VID:\s+${VID},.*SN: -> Record
-  ^PID:\s+${PID}.*,.*VID:\s+${VID},.*SN: -> Record
-  ^PID:\s+,.*VID:\s+${VID},.*SN:\s+${SN} -> Record
-  ^PID:\s+${PID}.*,.*VID:\s+${VID}.*
-  ^PID:\s+,.*VID:\s+${VID}.*
-  ^.*SN:\s+${SN} -> Record
-  ^.*SN: -> Record
-
+  ^NAME:\s+"${NAME}",\s+DESCR:\s+"${DESCR}"\s*$$
+  ^\s*Name:\s+${NAME}\s+Descr:\s+${DESCR}\s*$$
+  ^\s*PID:\s+${PID}\s*,\s+VID:\s+${VID}\s*,\s+SN:\s*${SN}$$ -> Record
+  ^\s*PID:\s+${PID}\s+VID:\s+${VID}\s+SN:\s+${SN}$$ -> Record
+  #
+  ^\s*$$ 
+  ^(Mon|Tue|Wed|Thu|Fri|Sat|Sun)\s\S{3}\s+\d
```

Comment:
I haven't looked at an XR in some time. Does this output also happen in a programatic interface with Netmiko/Napalm?